src/main/java/org/broadinstitute/hellbender/tools/gvs/common/CommonCode.java

```diff
@@ -44,9 +44,12 @@ public static List<Integer> getGTAlleleIndexes(final VariantContext variant) {
 
     public static String getGTString(final VariantContext variant) {
         List<Integer> allele_indices = getGTAlleleIndexes(variant);
-        if (allele_indices.size() != 2){
-            throw new IllegalArgumentException("GT doesnt have two alleles");
-        }
+
```

Comment:
Does this need to stay? Not a fan of commented out code

Reply:
I don't want the fact that we changed functionality to be lost, but I left it as a comment saying that instead of the old code showing that